examples/quickstart-xgboost/utils.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+
+
+def client_args_parser():
+    """Parse arguments to define experimental settings on client side."""
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--num_partitions", default=20, type=int, help="Number of partitions."
```

Comment:
We usually us `-` instead of `_` in command line arguments. Let's update all the args to keep this consistent:
```suggestion
        "--num-partitions", default=20, type=int, help="Number of partitions."
```

Reply:
hi @danieljanes, if i changed to `-`, it can't be recognised as a variable when referring it using `parser = argparse.ArgumentParser`. e.g. `parser.node-id`. i think it breaks their naming rules

Reply:
No worries, solved!